types/evidence.go

```diff
@@ -684,7 +688,11 @@ func NewMockDuplicateVoteEvidenceWithValidator(height int64, time time.Time,
 	vB := voteB.ToProto()
 	_ = pv.SignVote(context.Background(), chainID, vB)
 	voteB.Signature = vB.Signature
-	return NewDuplicateVoteEvidence(voteA, voteB, time, NewValidatorSet([]*Validator{val}))
+	ev, err := NewDuplicateVoteEvidence(voteA, voteB, time, NewValidatorSet([]*Validator{val}))
+	if err != nil {
+		panic(err)
```

Comment:
Since this isn't expected to happen, it would be helpful if we gave a more informative message in case it does. Perhaps, say:

```go
log.Panicf("Constructing duplicate vote evidence for mock: %v", err)
// or
panic("constructing duplicate vote evidence for mock: "+err.Error())
```